src/chigame/games/models.py

```diff
@@ -6,13 +6,93 @@
 
 class Game(models.Model):
     """
-    A game (Chess, Checkers, etc.)
+    A game like Chess, Checkers, etc.
     """
 
-    name = models.TextField()
+    # Basic information
+    name = models.CharField(max_length=255)
     description = models.TextField()
+    image = models.URLField(default="/static/images/no_picture_available.png")
```

Comment:
Might not be relevant for this particular PR, but are we going to store all game images in this folder and in the repo?

Reply:
No. Suppose we had thousands and thousands of images, it would start bloat the repo. 

I googled around and the recommendations I found are that  images should be hosted on either a separate server or  blob storage service.

Reply:
this would actually be a good thing to comment in the code itself! That way anyone looking at the game model for whatever reason will be able to see that there is a more durable longer term solution that can/eventually should be implemented